src/Functions/EnqueueVacancyAnalyticsSummaryGenerator.cs

```diff
@@ -7,33 +8,32 @@
 
 namespace Esfa.VacancyAnalytics.Functions
 {
-    public static class EnqueueVacancyAnalyticsSummaryGenerator
+    public class EnqueueVacancyAnalyticsSummaryGenerator
     {
-        private const string LocalSettingsFileName = "local.settings.json";
         private const string VacancyEventStoreConnStringKey = "VacancyAnalyticEventsSqlDbConnectionString";
         private const string QueueStorageConnStringKey = "QueueStorage";
+        private readonly ILogger<EnqueueVacancyAnalyticsSummaryGenerator> _log;
+        private readonly IConfiguration _config;
 
-        [FunctionName("EnqueueVacancyAnalyticsSummaryGenerator")]
-        public static async Task Run([TimerTrigger("0 1 * * * *")] TimerInfo myTimer, ILogger log, ExecutionContext context)
+        public EnqueueVacancyAnalyticsSummaryGenerator(ILogger<EnqueueVacancyAnalyticsSummaryGenerator> log, IConfiguration config)
         {
-            log.LogInformation($"C# Timer trigger {nameof(EnqueueVacancyAnalyticsSummaryGenerator)} function executed at: {DateTime.Now}");
+            _log = log;
+            _config = config;
+        }
 
-            var config = new ConfigurationBuilder()
-                .SetBasePath(context.FunctionAppDirectory)
-                .AddJsonFile(LocalSettingsFileName, optional: true, reloadOnChange: true)
-                .AddEnvironmentVariables()
-                .Build();
+        [FunctionName("EnqueueVacancyAnalyticsSummaryGenerator")]
+        public async Task Run([TimerTrigger("0 1 * * * *")] TimerInfo myTimer, ILogger log, ExecutionContext context)
+        {
+            log.LogInformation($"C# Timer trigger {nameof(EnqueueVacancyAnalyticsSummaryGenerator)} function executed at: {DateTime.UtcNow}");
 
-            var reader = new VacancyEventStoreReader(config.GetConnectionString(VacancyEventStoreConnStringKey), log);
+            var reader = new VacancyEventStoreReader(_config.GetConnectionString(VacancyEventStoreConnStringKey), log);
```

Comment:
Because the particular logger type can't be injected through DI. Will need to address that level of logging after. The functions apps don't like `Logger<Type>`.